nixos/doc/manual/development/settings-options.section.md

```diff
@@ -364,6 +364,67 @@ have a predefined type and string generator already declared under
     and returning a set with [CDN](https://github.com/dzikoysk/cdn)-specific
     attributes `type` and `generate` as specified [below](#pkgs-formats-result).
 
+`pkgs.formats.kdl` { }
+
+:   A function taking an empty attribute set (for future extensibility)
```

Comment:
you added the version like i asked, but forgot to update this part of the manual

```suggestion
`pkgs.formats.kdl` { `version` }

:   A function taking a version number (which must be exactly `1`, for future extensibility)
```